### DISCUSSION TITLE
Using unocss in Vuecli4 resulted in CSS background image not being found .

### DISCUSSION THREAD

**Discussion by mdxiaohu** [unocss/unocss#3146] [General]
Must be configured in Vue cli4 .

```
css: {
	extract: {
	    filename: '[name].[hash:9].css',
	}
}
```

Configuring this property will result in the inability to find image addresses imported through URLs in CSS .

For example, CSS like this .

`background: url("~@/assets/headerBg.svg")`

Resolved address .

`/static/css/static/img/headerBg.svg`

Correct address .

`/static/img/headerBg.svg`

What are the ways to avoid it .

I've been stuck with this attribute for a day now .

**Comment by asd8855** [comment]
+1,I have the same problem

**Reply by mdxiaohu** [reply]
Writing without background style is also acceptable.
I am currently encountering a new issue where '@ apply' cannot be used in Vuecli4 or cli5.
Without 'unocss' with' @ apply ', there is no soul .

**Reply by mdxiaohu** [reply]
这就奇了怪了，咋没见几个发个中文，非说English，让我很尴尬。
